Automattic/wp-calypso
# Jetpack UX: Change database error urls based on new docs

Fixes https://github.com/Automattic/dotcom-forge/issues/3710
## Proposed Changes

This PR changes the doc URL to point to the new docs for Jetpack database error.


## Testing Instructions

1. Create a WoA site. 
2. Use phpmyadmin to delete `wp_options` table.
3. Visit `/posts` page.
4. Make sure that you see the following message:
![database_errors_new](https://github.com/Automattic/wp-calypso/assets/497103/d73c9b53-f644-40de-8ac7-31324825c111)
5. Ensure that docs point [here](https://wordpress.com/support/resolve-jetpack-errors/#site-is-missing-database-tables)



## Pre-merge Checklist
